problems in both the source and translation cards

Notes
======
problems in both the source and target cards

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 153
UDID | a4b66484d349dee1
Android Release | 5.0.2
Android SDK | 21
Brand | lge
Device | e9wifi
Log history
======
```java
2/8/17 3:53 PM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo1_php_text_reg
2/8/17 3:52 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/aag_gal_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:52 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_mrk_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:52 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_tit_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:52 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_obs_text_obs/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:52 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:52 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_3jn_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:51 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/aag_gal_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:51 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_mrk_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:51 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_tit_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:51 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_obs_text_obs/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:51 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:51 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_3jn_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:50 PM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo1_php_text_reg
2/8/17 3:50 PM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo2_php_text_reg
2/8/17 3:47 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/aag_gal_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:47 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_mrk_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:47 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_tit_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:47 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_obs_text_obs/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:47 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:47 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_3jn_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:46 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/aag_gal_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:46 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_mrk_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:46 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_tit_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:46 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_obs_text_obs/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:46 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:46 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_3jn_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:42 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/aag_gal_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:42 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_mrk_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:42 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_tit_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:42 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_obs_text_obs/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:42 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:42 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_3jn_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:41 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/aag_gal_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:41 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_mrk_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:41 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_tit_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:41 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_obs_text_obs/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:41 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:41 PM  E/com.door43.translationstudio.services.BackupService: runBackup: Failed to commit changes before backing up
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1025)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:988)
	at com.door43.translationstudio.services.BackupService.runBackup(BackupService.java:119)
	at com.door43.translationstudio.services.BackupService.access$000(BackupService.java:32)
	at com.door43.translationstudio.services.BackupService$1.run(BackupService.java:63)
	at java.util.Timer$TimerImpl.run(Timer.java:284)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_3jn_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 6 more

2/8/17 3:38 PM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation aag_gal_text_reg
2/8/17 3:38 PM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_mrk_text_reg
2/8/17 3:38 PM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_tit_text_reg
2/8/17 3:38 PM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo2_obs_text_obs
2/8/17 3:38 PM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_php_text_ulb
2/8/17 3:38 PM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo2_3jn_text_reg
     

```

--- a/app/src/main/java/com/door43/translationstudio/ui/translate/ReviewModeAdapter.java
+++ b/app/src/main/java/com/door43/translationstudio/ui/translate/ReviewModeAdapter.java
@@ -827,11 +827,13 @@ public class ReviewModeAdapter extends ViewModeAdapter<ReviewModeAdapter.ViewHol
             @Override
             public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 if (isChecked) {
-                    // make sure to capture verse marker changes changes before dialog is displayed
-                    Editable changes = holder.mTargetEditableBody.getText();
-                    item.renderedTargetText = changes;
-                    String newBody = Translator.compileTranslation(changes);
-                    item.targetText = newBody;
+                    if(item.isEditing) {
+                        // make sure to capture verse marker changes changes before dialog is displayed
+                        Editable changes = holder.mTargetEditableBody.getText();
+                        item.renderedTargetText = changes;
+                        String newBody = Translator.compileTranslation(changes);
+                        item.targetText = newBody;
+                    }
 
                     new AlertDialog.Builder(mContext,R.style.AppTheme_Dialog)
                             .setTitle(R.string.chunk_checklist_title)

